### ISSUE TITLE
Download Minecraft experimental snapshot

### ISSUE THREAD

**Issue by hax0r31337** [gorilla-devs/GDLauncher#1043]
**Is your feature request related to a problem? Please describe.**
I want to play Minecraft 1.18 experimental snapshot with GDL

**Describe the solution you'd like**
Add a list of experimental versions in the download page

**Additional context**
![image](https://user-images.githubusercontent.com/65506006/130345795-3101555c-5c7c-49c0-a939-2658f221636e.png)

**Comment by hypherionmc** [comment]
Yes this is on our TODO list. Can't say when it will be available at this time
